integrationExamples/gpt/pbjs_video_adUnit.html

```diff
@@ -24,72 +24,72 @@
   <script src="/build/dev/prebid.js" async=true></script>
   <script>
 
-  var pbjs = pbjs || {};
-  pbjs.que = pbjs.que || [];
+      var pbjs = pbjs || {};
```

Comment:
I guess changes to `integrationExamples/gpt/pbjs_video_adUnit.html` are accidental?